### PR TITLE
[Currency Bot] Bumped prisma from 4.5.0 to 5.12.1

### PR DESCRIPTION
**Pull request by kirrg001**
Tada!

refs [INSTA-743](https://jsw.ibm.com/browse/INSTA-743)

### CODE REVIEW

**Comment by kirrg001** [comment]
v4 is affected as well.

< 4.10 works

4.10
https://github.com/prisma/prisma/commit/30ebd6a21b180cea10320228e0392f2a5de670b6

**Comment by kirrg001** [comment]
< 5.2 also does not work. They did not backport the fix from 5.2 to >= 4.10. We cant add url support for this version range.

**Comment by kirrg001** [comment]
We merge this as feature.

File: package.json [hunk 1]
```diff
@@ -196,7 +196,9 @@
     "pkg-up": "^3.1.0",
     "portfinder": "^1.0.32",
     "prettier": "^2.5.1",
-    "prisma": "^4.5.0",
+    "prisma": "^5.12.1",
+    "prisma-v4": "npm:prisma@4.16.2",
+    "prisma-v450": "npm:prisma@4.5.0",
```
Comment (kirrg001) [inline package.json:201]: 4.5 -> support for getConfig
4.16.2 -> broken (cant fix)
5.12.1 -> added new support for replacement of getConfig

Comment (kirrg001) [reply package.json:201]: See comments on PR body

**Comment by kirrg001** [comment]
@instana/eng-nodejs RFR

**Review by aryamohanan — APPROVED** [review]
LGTM

**Pull request merged**